src/components/ContextCard.vue

```diff
@@ -45,14 +46,50 @@ export default {
   },
   data: function () {
     return {
+      contextData: {},
       showDetails: true
     };
   },
-  computed: {
-
+  watch: {
+    'entry.contextCardUrl'(val){
+      this.getContextFile(val)
+    }
   },
   methods: {
+    getContextFile: function (contextFileUrl) {
+      fetch(contextFileUrl)
+        .then((response) =>{
+          if (!response.ok){
+            throw Error(response.statusText)
+          } else {
+             return response.json()
+          }
+        })
+        .then((data) => {
+          this.contextData = data
+          console.log(this.contextData)
+        })
+        .catch(() => {
+          //set defaults if we hit an error
+          this.thumbnail = require('@/../assets/missing-image.svg')
+          this.discoverId = undefined
+        });
+    },
+    getFileFromPath: function(path){
+      return  `${this.entry.apiLocation}s3-resource/${this.entry.discoverId}/${this.entry.version}/files/${path}`
+    },
+    openViewFile: function(view){
+
+      // note that we assume that the view file is in the same directory as the scaffold (viewUrls take relative paths)
+      this.entry.viewUrl = view.path.split('/')[view.path.split('/').length-1]
+      this.entry.type = 'Scaffold View'
+      EventBus.$emit("PopoverActionClick", this.entry)
+    }
+    
   },
+  mounted: function(){
```

Comment:
Can try to get the context file at create, and make sure the watch is entry at watch is set to immediate.